pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy

```diff
@@ -106,12 +106,18 @@ public class Converter {
         CompilationUnit cu = new CompilationUnit(
             makeCompilerConfiguration(),
             new CodeSource(src, new Certificate[0]),
-            new GroovyClassLoader());
+            getCompilationClassLoader());
         cu.addSource(src);
 
         return compilationUnitToPipelineDef(cu)
     }
 
+    private static GroovyClassLoader getCompilationClassLoader() {
+        Jenkins j = Jenkins.instance
```

Comment:
Durrr. Good point. `Jenkins.instance` is `Jenkins.getInstance()` (thanks, Groovy magic!), but there's no need for the null check.